test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -143,6 +143,37 @@ def options_with_apply_defaults(v):
     return v
 
 
+def check_removal_version(v, version_name, collection_name_name, error_code='invalid-removal-version'):
+    version = v.get(version_name)
+    collection_name = v.get(collection_name_name)
```

Comment:
```suggestion
def check_removal_version(v, version_name, collection_name, error_code='invalid-removal-version'):
    version = v.get(version_name)
    collection_name = v.get(collection_name)
```
could we use the replacement safely everywhere or anything else instead of `collection_name_name`?

Reply:
`collection_name_name` is the *name* of the field which is populated into `collection_name`. It is not a typo/mistake.

Reply:
I understand that it is not a typo, just looks strange

Reply:
I could also rename `version_name`, `collection_name_name` to `version_field`, `collection_name_field` - would that be better? Or another suffix maybe?

Reply:
`would that be better?` yes, it would look very good:)

Reply:
Fixed in 25542c679f25b19c80ee0a8dd5825311a994d491.